src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { arrayOf, bool, node } from 'prop-types';
+import glamorous from 'glamorous';
+
+import Button from '../Button/Button';
+import constants from '../../constants';
+
+const { borderRadius } = constants;
+const { Div } = glamorous;
+
+const ButtonGroupItem = glamorous(Button)({
+   borderRadius: 0,
+   borderRightWidth: 0,
+
+   '&:first-of-type': {
+      borderTopLeftRadius: borderRadius,
+      borderBottomLeftRadius: borderRadius,
+   },
+   '&:last-of-type': {
+      borderTopRightRadius: borderRadius,
+      borderBottomRightRadius: borderRadius,
+      borderRightWidth: 1,
+   },
+});
+
+const ButtonGroup = ({ children, disabled, ...props }) => (
+   <Div display="inline-block" {...props}>
+      {React.Children.map(children, child => {
+         if (child.type !== Button) {
```

Comment:
I'm a little bit confused why this is necessary. Is there really a good reason to ever wrap something that isn't a `Button` in a `ButtonGroup`?

Reply:
In my use case I have a hidden input inside the ButtonGroup so there's one.